test/test_transforms_tensor.py

```diff
@@ -608,6 +511,88 @@ def test_to_grayscale(device, Klass, meth_kwargs):
     )
 
 
+@cpu_only
+@pytest.mark.xfail()
+@pytest.mark.parametrize(
+    'in_dtype,out_dtype', [
+        (int_dtypes() + float_dtypes(), int_dtypes() + float_dtypes())
+    ]
+)
+def test_convert_image_dtype(in_dtype, out_dtype):
+    tensor, _ = _create_data(26, 34, device="cpu")
+    batch_tensors = torch.rand(4, 3, 44, 56, device="cpu")
+
+    in_tensor = tensor.to(in_dtype)
+    in_batch_tensors = batch_tensors.to(in_dtype)
+
+    fn = T.ConvertImageDtype(dtype=out_dtype)
+    scripted_fn = torch.jit.script(fn)
+
+    if (in_dtype == torch.float32 and out_dtype in (torch.int32, torch.int64)) or \
+            (in_dtype == torch.float64 and out_dtype == torch.int64):
+        with pytest.raises(RuntimeError, match=r"cannot be performed safely"):
+            _test_transform_vs_scripted(fn, scripted_fn, in_tensor)
+        with pytest.raises(RuntimeError, match=r"cannot be performed safely"):
+            _test_transform_vs_scripted_on_batch(fn, scripted_fn, in_batch_tensors)
+
+    _test_transform_vs_scripted(fn, scripted_fn, in_tensor)
+    _test_transform_vs_scripted_on_batch(fn, scripted_fn, in_batch_tensors)
+
+    with get_tmp_dir() as tmp_dir:
+        scripted_fn.save(os.path.join(tmp_dir, "t_convert_dtype.pt"))
+
+
+@cpu_only
+@pytest.mark.parametrize(
+    'policy,fill', [
+        ([policy for policy in T.AutoAugmentPolicy], [None, 85, (10, -10, 10), 0.7, [0.0, 0.0, 0.0], [1, ], 1])
+    ]
+)
```

Comment:
Since there are 2 for loops (one inside the other), we'll need 2 parametrizations here:
```py
@pytest.mark.parametrize('policy', [policy for policy in T.AutoAugmentPolicy])
@pytest.mark.parametrize('fill', [None, 85, (10, -10, 10), 0.7, [0.0, 0.0, 0.0], [1, ], 1])

Reply:
Thanks, I did this.